pkg/mcp/loader.go

```diff
@@ -70,10 +75,16 @@ func NewSessionManager(ctx context.Context, defaultLoader *gmcp.Local, opts Opti
 		})); err != nil {
 			log.Warnf("failed to create MCP namespace, namespace must exist for MCP deployments to work: %v", err)
 		}
+
+		clientset, err = kubernetes.NewForConfig(config)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Sadly, the controller-runtime client that we typically use can't really interact with events, so we had to do a clientset here just for that.

Reply:
You meant logs, right? Just making sure I understand the issue here.

Reply:
Logs too I think, but I did mean events. I don't think I really checked for logs.

Reply:
Sorry, my memory was incorrect. Yes, I was able to use the main controller-runtime client to handle the events and only wound up needing the clientset for logs.

When I was originally working on events, AI told me I needed to use the clientset, but I figured out a way to not use it, but then when it came to the logs I realized I did need to use it there. Then when responding here I forgot the details of that and thought I was using it for events too, when I wasn't.